README.md

```diff
@@ -1,12 +1,98 @@
-# heat_pump_suitability
+# asf_heat_pump_suitability
+
+Calculate heat pump suitability scores for lower-layer super output areas (LSOAs) in England and Wales using domestic
+EPC data and supplementary sources. Scores are weight-adjusted for LSOAs where possible to reduce bias.
+Intermediate outputs include:
+
+- EPC data weighted according to LSOA using Iterative Proportional Fitting to reduce bias
+- EPC data enhanced with new features including: lat/lon; listed building status; building conservation zone status;
+  off gas status; average garden size per MSOA; property density per LSOA
+- Individual garden size estimates for UPRNs in EPC
 
 ## Setup
```

Comment:
Think it'd be good to put this section at the end, just before the contributor guidelines section. It's a bit off putting for anyone coming to the repo to only look at the code.